INSTALL.md

```diff
@@ -0,0 +1,68 @@
+# Install Tomateos
+### Required dependencies
+
+Required dependencies trough Packet-Managers/Manual installation:
+* Ruby >= 2.3.3
+* Ruby-Gems
+* MongoDB (More installation instructions at: https://docs.mongodb.com/manual/administration/install-on-linux/ for your desired distribution)
+
+If you wan't to install it with RVM, you will need RVM of course too.
+##### Install ruby with RVM(works with nearly all distros)
+
+Install RVM:
+```
+su
+$PackageManager install rvm
```

Comment:
Heh, yeah, that was the intention. I wrote that down fast to get that done too before i open a PR. It's on the ToDo for tomorrow.